server/src/rest/implementation/converter/SessionB2RConverter.ts

```diff
@@ -0,0 +1,13 @@
+import BoundarySessionOutput from "../../api/entity/BoundarySessionOutput";
+
+export default class SessionB2RConverter {
+  processData(data: BoundarySessionOutput): object {
```

Comment:
gerai būtų susikurti klasę ar interface RestSessionOutput, turėsi testuose ir ten kur naudoja typesafety

Reply:
Klasę su field be `private` ( tik readonly)